### PR TITLE
Homework21: Marie Ambrazhei 

### PR DESCRIPTION
**Pull request by MarieAmbrazhei**

### CODE REVIEW

**Comment by MarieAmbrazhei** [comment]
Sergey, I added a report which I invoke with the command pytest homework21/tests/test_library_functionality.py --html-log-level=WARNING.
This way you can check the log level in the HTML file.

File: .pylintrc
Comment (Stml89) [inline .pylintrc:1]: Since you are using `pylint.yaml`, you can move those rules to it and delete this file

File: mypy.ini
Comment (Stml89) [inline mypy.ini:1]: Add it to `mypy.yaml` and remove this file

File: homework21/tests/test_library_functionality.py [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+
+@pytest.fixture
+def user_2() -> User:
+    """
+    Creates and deletes user_2
+    """
+    user = User('user_id_2')
+    yield user
+    del user
```
Comment (Stml89) [inline homework21/tests/test_library_functionality.py:45]: if you have `conftest.py` move fixture to this file

File: homework21/tests/test_library_functionality.py [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+
+
+def test_take_book_positive(user_1):
```
Comment (Stml89) [inline homework21/tests/test_library_functionality.py:48]:
```suggestion
def test_take_book_positive(user_1, setup_books):
```

File: homework21/tests/test_library_functionality.py [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+    """Test taking a book that is available."""
+    log.info('User tries to take book with 134527 isbn')
+    assert user_1.take_book('134527'), ('Failed to take the book with'
```
Comment (Stml89) [inline homework21/tests/test_library_functionality.py:51]: and use book from `setup_book` fixture

File: homework21/tests/test_bank_functionality.py [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+    """
+    bank_1 = initial_bank
+    dep_2 = deposit_2
+    expected_amount = 18305.86442063339
+    result = bank_1.deposit(dep_2)
+    log.info(f'Testing deposit_2: expected {expected_amount}, got {result}')
+    assert round(result, 2) == round(expected_amount, 2), (
+        f'Result {result} does not match the expected amount {expected_amount}'
+    )
+    log.info('Successfully verified deposit calculation')
```
Comment (Stml89) [inline homework21/tests/test_bank_functionality.py:66]: those 2 tests can be combine into 1 with `pytest.mark.parametrize`

**Review by Stml89 — CHANGES_REQUESTED** [review]

File: .pylintrc
Comment (MarieAmbrazhei) [reply .pylintrc:1]: done

File: mypy.ini
Comment (MarieAmbrazhei) [reply mypy.ini:1]: done

File: homework21/tests/test_library_functionality.py (hunk 1 above)
Comment (MarieAmbrazhei) [reply homework21/tests/test_library_functionality.py]: done

File: homework21/tests/test_library_functionality.py (hunk 2 above)
Comment (MarieAmbrazhei) [reply homework21/tests/test_library_functionality.py]: done

File: homework21/tests/test_library_functionality.py (hunk 3 above)
Comment (MarieAmbrazhei) [reply homework21/tests/test_library_functionality.py]: done!

File: homework21/tests/test_bank_functionality.py (hunk 4 above)
Comment (MarieAmbrazhei) [reply homework21/tests/test_bank_functionality.py]: done

**Review by Stml89 — APPROVED** [review]

**Pull request merged**